crowdin.yml

```diff
@@ -0,0 +1,16 @@
+project_id_env: _CROWDIN_PROJECT_ID
+api_token_env: CROWDIN_API_TOKEN
```

Comment:
⛏️ Two different variable formats (leading underscore).

Reply:
`_CROWDIN_PROJECT_ID` is a job level environment variable versus a step level. On smaller workflows, this pattern is marginal, but on workflows with 500-1000 lines, the differentiation helps with maintenance.

The workflow linter rule can be found here: https://github.com/bitwarden/workflow-linter/blob/f204a10c9bb6e623f8ab2223a9b577bcf94f46c1/src/bitwarden_workflow_linter/rules/job_environment_prefix.py#L19